test/get-all-rep-allowances.js

```diff
@@ -0,0 +1,52 @@
+const assert = require("chai").assert;
```

Comment:
Are you able to get this test file to run as it is?  I get the following error:
```
/home/jack/src/rep-migration-scripts-copy/test/get-all-rep-allowances.js:10
const LegacyRepToken = artifacts.require("LegacyRepToken");
                       ^

ReferenceError: artifacts is not defined
    at Object.<anonymous> (/home/jack/src/rep-migration-scripts-copy/test/get-all-rep-allowances.js:10:24)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at /home/jack/src/rep-migration-scripts-copy/node_modules/mocha/lib/mocha.js:231:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (/home/jack/src/rep-migration-scripts-copy/node_modules/mocha/lib/mocha.js:228:14)
    at Mocha.run (/home/jack/src/rep-migration-scripts-copy/node_modules/mocha/lib/mocha.js:514:10)
    at Object.<anonymous> (/home/jack/src/rep-migration-scripts-copy/node_modules/mocha/bin/_mocha:480:18)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Function.Module.runMain (module.js:684:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
```
`truffle compile` doesn't even work currently (probably due to Solidity versioning issues?):
```
$ npx truffle compile
Compiling ./contracts/Migrations.sol...
Compiling ./contracts/RepToken.sol...
Compiling ./contracts/helpers/InitializableTester.sol...
Compiling ./contracts/helpers/LegacyRepToken.sol...
Compiling zeppelin-solidity/contracts/lifecycle/Pausable.sol...
Compiling zeppelin-solidity/contracts/math/SafeMath.sol...
Compiling zeppelin-solidity/contracts/ownership/Ownable.sol...
Compiling zeppelin-solidity/contracts/token/BasicToken.sol...
Compiling zeppelin-solidity/contracts/token/ERC20.sol...
Compiling zeppelin-solidity/contracts/token/ERC20Basic.sol...
Compiling zeppelin-solidity/contracts/token/PausableToken.sol...
Compiling zeppelin-solidity/contracts/token/StandardToken.sol...

/home/jack/src/rep-migration-scripts-copy/contracts/RepToken.sol:94:3: TypeError: Override changes extended function signature.
  function unpause() onlyOwner whenPaused afterInitialized returns (bool) {
  ^
Spanning multiple lines.
Compiliation failed. See above.
```

Reply:
FWIW, I don't think it's necessary to have the tests working in the format of this test file / using truffle.  What sanity checks would be useful to do, to make sure we've actually got all the allowances?  e.g., with rep-holder address balances, you can check that all the balances summed together are 11,000,000 REP.

Reply:
The migration script in augur-core validates the allowances before moving balances so unless they are 1:1 with the chain we'll never complete migration.

If the worry is the allowances collected from the script will be wrong I don't think there is a really clean way to validate it other than just doing a 1:1 check.

Would not surprise me if the test is no longer working. I had to flail a lot to get it to run at all and contracts to compile with truffle. Let me know what you think we should add or delete here and I'll pick it up when the MVP queue is empty.